game.rb

```diff
@@ -0,0 +1,39 @@
+require_relative 'environment.rb'
+
+class Game
+    attr_accessor :board, :depot, :control
+
+    def initialize
+        @board = {
+            players: {},
+            control: {},
+            status: nil
+        }
```

Comment:
I would define these kind of hashes using one line, like: 
`@board = { players: {}, control: {}, status: nil }`